apps/server/src/modules/user-login-migration/controller/api-test/user-login-migration.api.spec.ts

```diff
@@ -151,6 +152,186 @@ describe('UserLoginMigrationController (API)', () => {
 		});
 	});
 
+	describe('[POST] /start', () => {
+		describe('when current User start the migration successfully', () => {
+			const setup = async () => {
+				const sourceSystem: System = systemFactory.withLdapConfig().buildWithId({ alias: 'SourceSystem' });
```

Comment:
The alias is a freeform field, so setting it like this is like it would be done in prod (although currently we only provide it via DB injection)

Will rename the class :)

Reply:
Yes the renames already exists in arc tickets. It include also BaseFactory to BaseTestFactory.

Reply:
yes